axelrod/strategies/cycler.py

```diff
@@ -72,9 +72,11 @@ def __init__(self, cycle="CCD") -> None:
         self.classifier['memory_depth'] = len(cycle) - 1
 
     def strategy(self, opponent: Player) -> Action:
+        actions = {'C': Actions.C, 'D': Actions.D}
```

Comment:
this is where i changed Cycler so that it no longer assumes Actions.C = 'C' and Actions.D = 'D'